app/helpers/measure_unit_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MeasureUnitHelper
```

Comment:
Is this view helper code? Feels more like its building a data structure? - Would it make more sense as a module a `services/concerns` folder?